src/entities/amm/voltz-config/config.ts

```diff
@@ -0,0 +1,244 @@
+import { NetworkConfiguration } from './types';
```

Comment:
Copied from UI.

Reply:
What's this used for?

Reply:
To the whitelists and so. It's on the UI rn but we'd like to move it.